tests/cli/test_rasa_data.py

```diff
@@ -144,6 +152,415 @@ async def mock_from_importer(importer: TrainingDataImporter) -> Validator:
         data.validate_files(args)
 
 
+@pytest.mark.parametrize(
+    "config_file, nlu_train_file, nlu_test_file, paraphrases_file, classification_report_file, "
+    "min_paraphrase_sim_score, max_paraphrase_sim_score, augmentation_factor, intent_proportion",
+    [
+        (
+            "config.yml",
+            "nlu_train.yml",
+            "nlu_test.yml",
+            "paraphrases.yml",
+            "nlu_classification_report_no_augmentation.json",
+            0.1,
+            -1.0,
+            2,
+            0.5,
+        ),
+        (
+            "config.yml",
+            "nlu_train.yml",
+            "nlu_test.yml",
+            "paraphrases.yml",
+            "nlu_classification_report_no_augmentation.json",
+            0.1,
+            1.0,
+            -2,
+            0.5,
+        ),
+        (
+            "config.yml",
+            "nlu_train.yml",
+            "nlu_test.yml",
+            "paraphrases.yml",
+            "nlu_classification_report_no_augmentation.json",
+            0.1,
+            1.0,
+            -2,
+            -0.5,
+        ),
+        (
+            "config.yml",
+            "nlu_train.yml",
+            "nlu_test.yml",
+            "paraphrases.yml",
+            "nlu_classification_report_no_augmentation.json",
+            0.1,
+            1.0,
+            -2,
+            1.5,
+        ),
+    ],
+)
+def test_rasa_data_augment_nlu_invalid_parameters(
+    config_file: Text,
+    nlu_train_file: Text,
+    nlu_test_file: Text,
+    paraphrases_file: Text,
+    classification_report_file: Text,
+    min_paraphrase_sim_score: float,
+    max_paraphrase_sim_score: float,
+    augmentation_factor: float,
+    intent_proportion: float,
+):
+    tmp_path = rasa.utils.io.create_temporary_directory()
+    out_path = os.path.join(tmp_path, "augmentation_results")
+    os.makedirs(out_path)
```

Comment:
can we please use the `tmp_path` fixture for this to make sure the temporary directory is cleaned up after the tests?